crates/ruff_linter/src/rules/airflow/snapshots/ruff_linter__rules__airflow__tests__AIR302_AIR302_context.py.snap

```diff
@@ -0,0 +1,109 @@
+---
+source: crates/ruff_linter/src/rules/airflow/mod.rs
+snapshot_kind: text
+---
+AIR302_context.py:9:30: AIR302 `execution_date` is removed in the Airflow context in Airflow 3.0
```

Comment:
I'm not sure whether we really need to change the code to just add `Airflow context` here 🤔 we probably could just use message

Reply:
Changed it